hazelcast-jet-distribution/src/bin-filemode-755/jet-start

```diff
@@ -2,14 +2,30 @@
 
 SCRIPT_DIR=$(dirname "$0")
 #shellcheck source=bin-regular/common.sh
+
+if [[ "$1" = "--help" ]] || [[ "$1" = "-h" ]]; then
+    echo "Usage: jet-start [-d]"
+    echo "  -d, --daemon   Starts Hazelcast Jet in daemon mode"
+    exit 0
+elif [[ "$1" = "-d" ]] || [[ "$1" = "--daemon" ]]; then
+    DAEMON="true"
+fi
+
 . "$SCRIPT_DIR"/common.sh
 
 echo "########################################"
 echo "# JAVA=$JAVA"
 echo "# JAVA_OPTS=${JAVA_OPTS_ARRAY[*]}"
 echo "# CLASSPATH=$CLASSPATH"
 echo "########################################"
-echo "Starting Hazelcast Jet"
 
-set -x
-$JAVA "${JAVA_OPTS_ARRAY[@]}" -cp "$CLASSPATH" com.hazelcast.jet.server.JetMemberStarter
+if [[ "$DAEMON" = "true" ]]; then
+    mkdir -p $JET_HOME/logs
+    JET_LOG="$JET_HOME/logs/hazelcast-jet.out"
+    echo "Starting Hazelcast Jet in daemon mode. Standard out and error will be written to $JET_LOG"
+    nohup $JAVA "${JAVA_OPTS_ARRAY[@]}" -cp "$CLASSPATH" com.hazelcast.jet.server.JetMemberStarter > "$JET_LOG" 2>&1 &
```

Comment:
we can get rid off specific log forwarding
`nohup $JAVA "${JAVA_OPTS_ARRAY[@]}" -cp "$CLASSPATH" com.hazelcast.jet.server.JetMemberStarter &`
should be enough thanks to log4j configuration. This eliminates above naming issue too

Reply:
I think if you turn this off and there's some error on startup i.e. node fails to start, then it won't be logged. I didn't find a good solution to this though, Kafka does something similar.